nservicebus/hosting/cloud-services-host/index.md

```diff
@@ -21,145 +21,45 @@ The Azure Platform and NServiceBus make a perfect fit. On the one hand the Azure
 
 If real scale is required (eg in tens, hundreds or even thousands of machines hosting each endpoint) than cloud services is the required deployment model.
 
-NOTE: If this level of scale is not required, there are [other hosting options available](hosting-options.md).
-
-
 ## Cloud Services - Worker Roles
 
 Reference the assembly that contains the Azure role entry point integration. The recommended way of doing this is by adding a NuGet package reference to the `NServiceBus.Hosting.Azure` package to the project.
 
-NOTE: If self hosting, done later in this article for Web Roles, everything can be configured using the configuration API and the extension methods found in the `NServiceBus.Azure` package, no need to reference the hosting package in that case.
+NOTE: If self hosting, done later in this article for Web Roles, everything can be configured using the configuration API and the extension methods found in the nservicebus azure related packages, no need to reference the hosting package in that case.
```

Comment:
> If self hosting, done later in this article for Web Roles, everything can be configured using the configuration API and the extension methods found in the nservicebus azure related packages, no need to reference the hosting package in that case.

When self-hosting everything can be configured using code API and extension methods available in the NServiceBus Azure related packages. In such a case it's not required to reference the hosting package in the project. Self-hosting is described later in this article.